include/uncertainty_planning_core/uncertainty_contact_planning.hpp

```diff
@@ -141,6 +140,133 @@ class UncertaintyPolicyExecutionResult
   }
 };
 
+#if UNCERTAINTY_PLANNING_CORE__SUPPORTED_ROS_VERSION == 2
+class ExecutionTimeLimit
+{
+public:
+  ExecutionTimeLimit(const double seconds, rclcpp::Clock::SharedPtr clock)
+    : seconds_(seconds), clock_(clock)
+  {
+    if (seconds_ < 0.0)
+    {
+      throw std::invalid_argument("Time limit cannot be less than zero seconds");
+    }
+    if (!clock_)
+    {
+      throw std::invalid_argument("No valid clock supplied");
+    }
+  }
+
+  explicit ExecutionTimeLimit(const double seconds)
+    : ExecutionTimeLimit(seconds, std::make_shared<rclcpp::Clock>())
+  {
+  }
+
+  double Seconds() const { return seconds_; }
+
+  rclcpp::Clock& Clock() { return *clock_; }
+
+private:
+  double seconds_;
+  rclcpp::Clock::SharedPtr clock_;
+};
+#elif UNCERTAINTY_PLANNING_CORE__SUPPORTED_ROS_VERSION == 1
+class ExecutionTimeLimit
+{
+public:
+  explicit ExecutionTimeLimit(const double seconds) : seconds_(seconds)
+  {
+    if (seconds_ < 0.0)
+    {
+      throw std::invalid_argument("Time limit cannot be less than zero seconds");
+    }
+  }
+
+  double Seconds() const { return seconds_; }
+private:
+  double seconds_;
+};
+#endif
+
+inline std::ostream& operator<<(
+  std::ostream& strm, const ExecutionTimeLimit& time_limit)
+{
+  return strm << time_limit.Seconds();
+}
+
+class ExecutionTimer {
+private:
+  template<typename T>
+  using OwningMaybe = common_robotics_utilities::OwningMaybe<T>;
+
+public:
+  explicit ExecutionTimer(const ExecutionTimeLimit& time_limit)
+    : time_limit_(time_limit)
```

Comment:
Please put brackets at the end of the line
```
explicit ExecutionTimer(const ExecutionTimeLimit& time_limit)
    : time_limit_(time_limit) {}
```

Reply:
See https://github.com/calderpg/uncertainty_planning_core/pull/3/commits/cf8ce915f230816208852918241ed1d6227c0d35.